src/power_grid_model/core/index_integer.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+"""
+Definition of integers used by the calculation core
+"""
+
+# define internal index integer
+from ctypes import c_int32, c_int64
+
+import numpy as np
+
+# pylint: disable=C0103
```

Comment:
Please use `invalid-name`.

Reply:
On the other hand... The name is invalid... It's a global definition, so it should actually be `IDX_C`.